src/lat/lattice-functions-transition-model.h

```diff
@@ -0,0 +1,96 @@
+// lat/lattice-functions-transition-model.h
+
+// Copyright 2009-2012   Saarland University (author: Arnab Ghoshal)
+//           2012-2013   Johns Hopkins University (Author: Daniel Povey);
+//                       Bagher BabaAli
+//                2014   Guoguo Chen
+
+// See ../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef KALDI_LAT_LATTICE_FUNCTIONS_TRANSITION_MODEL_H_
+#define KALDI_LAT_LATTICE_FUNCTIONS_TRANSITION_MODEL_H_
+
+#include <vector>
+#include <map>
```

Comment:
This should be alpha-sorted:
```suggestion
#include <map>
#include <vector>
```
The next block of includes, too.